packages/ui/react-ui/__tests__/CrossmintNFTDetail.spec.tsx

```diff
@@ -2,9 +2,11 @@ import "@testing-library/jest-dom";
 import { render, screen } from "@testing-library/react";
 import React from "react";
 
+import { NFT } from "@crossmint/client-sdk-base";
+
 import { CrossmintNFTDetail } from "../src/CrossmintNFTDetail";
 
-const nft = { chain: "ethereum", address: "0x12345", tokenId: "12" };
+const nft: NFT = { chain: "ethereum", contractAddress: "0x12345", tokenId: "12" };
```

Comment:
in the future would be great to have some sorta constraint either within typescript or with zod or something that contractAddress should be 42 chars long

Reply:
true, typings can evolve from here.